form/database/seeders/ProductCollection.php

```diff
@@ -5,7 +5,7 @@
 use Illuminate\Database\Console\Seeds\WithoutModelEvents;
 use Illuminate\Database\Seeder;
 use Illuminate\Support\Facades\DB;
-
+use App\Models\Product;
```

Comment:
@Yashbanu why do we need this import ?

Reply:
Initially I thought of using Eloquent to insert data so i used import later I used DB facade to insert data directly . It is not necessary now , I will remove it.

Reply:
Okay